apidef/oas/schema/x-tyk-api-gateway.json

```diff
@@ -958,6 +958,17 @@
       "required": [
         "enabled"
       ]
+    },
+    "X-Tyk-CustomDomain": {
+      "type": "object",
+      "properties": {
+        "name": "string",
```

Comment:
shall we add a format to the string `name` as `hostname` - https://json-schema.org/understanding-json-schema/reference/string.html#hostnames